austin_tui/__main__.py

```diff
@@ -33,6 +33,10 @@
 from austin_tui.view.austin import AustinProfileMode, AustinView
 from psutil import Process
 
+try: 
+    get_all_tasks = asyncio.all_tasks # Python 3.7+
+except AttributeError:
+    get_all_tasks = asyncio.Task.all_tasks # Python 3.6 compatibility
```

Comment:
The project uses Black as the default formatter so could you please:
- leave two blank lines before and after this block
- leave two spaces before starting an inline comment

Could you also rename `get_all_tasks` into `_get_all_tasks` as this is just a local alias.

Cheers!